modules/processing/static.py

```diff
@@ -345,9 +365,17 @@ def run(self):
         results["pe_timestamp"] = self._get_timestamp()
         results["pdb_path"] = self._get_pdb_path()
         results["signature"] = self._get_signature()
+
+        sigverification = self._get_signature_verification()
+        if sigverification is not None:
+            results["signature_verification_verified"] = sigverification['verified']
+            results["signature_verification_output"] = sigverification["output"].replace('\r','')
+
```

Comment:
Would be better to merge with the `"signature"` entry. E.g., `results["signature"]["digisign"] = ...` and `..["verified"] = True/False`.